demo/conjur-plugin.yml

```diff
@@ -0,0 +1,14 @@
+- hosts: localhost
+  tasks:
+    - name: add container to inventory
+      add_host:
+        name: ansible-lookup-target
+        ansible_connection: docker
+      changed_when: false
+
+- hosts: ansible-lookup-target
+  tasks:
+    - name: Write secret to file
+      vars:
+          super_secret_key: "{{lookup('conjur_variable', 'ansible-master-password')}}"
+      shell: echo "lookup('conjur_variable', 'ansible-master-password')" = {{super_secret_key}} > ./conjur_variable.txt
```

Comment:
you should be able to use the `file` module or similar here.

`shell -> echo 'something' -> write to file` is a code smell in ansible, there's often a module that does it in a cleaner more idempotent way.